server/stream.go

```diff
@@ -3249,9 +3252,9 @@ func (mset *stream) processInboundSourceMsg(si *sourceInfo, m *inMsg) bool {
 	var err error
 	// If we are clustered we need to propose this message to the underlying raft group.
 	if node != nil {
-		err = mset.processClusteredInboundMsg(m.subj, _EMPTY_, hdr, msg)
+		err = mset.processClusteredInboundMsg(m.subj, _EMPTY_, hdr, msg, nil)
 	} else {
-		err = mset.processJetStreamMsg(m.subj, _EMPTY_, hdr, msg, 0, 0)
+		err = mset.processJetStreamMsg(m.subj, _EMPTY_, hdr, msg, 0, 0, nil)
```

Comment:
Since we pass nil most of the time for last arg, keep same and have that call into new func that takes extra arg?

Reply:
Actually, out of 6 calls, 4 pass an mt object, so I decided not to. Let me know if you still want me to add a "processJetStreamMsgWithTrace()` or something like that.

Reply:
Your call, was a suggestion.